server/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "engines": {
+    "node": "8.1.1",
+    "npm": "5.0.3"
+  },
+  "scripts": {
+    "start": "node index.js",
+    "dev": "nodemon index.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "bcrypt-nodejs": "0.0.3",
+    "body-parser": "^1.18.2",
+    "connect-ensure-login": "^0.1.1",
```

Comment:
great to see you using additional modules. Is this package what's generating these server msgs:
![screen shot 2018-04-08 at 1 50 16 am](https://user-images.githubusercontent.com/18152993/38463745-3e568448-3acf-11e8-90d8-f238692bf47e.png)
I didn't see it described in the spec, but after reading over your program, I am stumped as to how you're making this happen - and I think it's a really nice touch!

Teach me, plz?

Reply:
https://http.cat/302

![Found!](https://http.cat/302)